mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestObjects.java

```diff
@@ -274,7 +272,10 @@ public Response getObjectVersions(@Parameter(example = "mir_mods_00004711") @Pat
         throws IOException {
         long modified = MCRXMLMetadataManager.instance().getLastModified(id);
         if (modified < 0) {
-            throw new NotFoundException("MCRObject " + id + " not found");
+            throw MCRErrorResponse.fromStatus(Response.Status.NOT_FOUND.getStatusCode())
+                .withErrorCode("MCROBJECT_NOT_FOUND")
```

Comment:
Would it make sense to collect all error codes as static variables in a central class?